test/E2E/EndToEndTest.cs

```diff
@@ -87,14 +91,17 @@ public void TestDotnetForceIncrementalUnsafe()
             buildCommand.Execute().Should().Pass();
             TestOutputExecutable(OutputDirectory, buildCommand.GetOutputExecutableName());
 
-            var latestWriteTimeFirstBuild = GetLastWriteTimeOfDirectoryFiles(OutputDirectory);
+            var binariesOutputDirectory = GetBinariesOutputDirectory(OutputDirectory, false);
```

Comment:
I moved this test method into https://github.com/dotnet/cli/blob/master/test/Microsoft.DotNet.Tools.Builder.Tests/IncrementalTests.cs#L28